batch7crud-roll8/src/main/java/com/lftechnology/batch7crud/controller/EmployeesController.java

```diff
@@ -0,0 +1,206 @@
+      request.setAttribute(AttributeConstants.EMPLOYEE, employee);
+      RequestDispatcher dispatcher = request.getRequestDispatcher(request.getContextPath() + UrlConstants.EMPLOYEE_EDIT_PAGE);
+      dispatcher.forward(request, response);
+    }
+    catch (Exception e) {
+      LOGGER.log(Level.SEVERE, e.getMessage(), e);
+      errorPage(request, response, e.getMessage());
+    }
+  }
+
+  private Employee setEmployeeAttributes (HttpServletRequest request) {
+    String name = request.getParameter(AttributeConstants.NAME);
+    String address = request.getParameter(AttributeConstants.ADDRESS);
+    String designation = request.getParameter(AttributeConstants.DESIGNATION);
+    String phone = request.getParameter(AttributeConstants.PHONE);
+
+    Employee employee = new Employee();
+    employee.setName(name);
+    employee.setAddress(address);
+    employee.setDesignation(designation);
+    employee.setPhone(phone);
+    return employee;
+  }
+
+  private void createProcess(HttpServletRequest request, HttpServletResponse response) {
+    try {
+      Employee employee = setEmployeeAttributes(request);
+      employeeService.create(employee);
+      response.sendRedirect(request.getContextPath() + UrlConstants.EMPLOYEE_ROUTE);
+    } catch (Exception e) {
+      LOGGER.log(Level.SEVERE, e.getMessage(), e);
+      errorPage(request, response, e.getMessage());
+    }
+  }
+
+  private void editProcess(HttpServletRequest request, HttpServletResponse response) {
+    try{
+      int id = parameterValueAsInt(request, 2);
+      Employee employee = setEmployeeAttributes(request);
+      employee.setId(id);
+      employeeService.edit(employee);
+      response.sendRedirect(request.getContextPath() + UrlConstants.EMPLOYEE_ROUTE);
+    }
+    catch (Exception e) {
+      LOGGER.log(Level.SEVERE, e.getMessage(), e);
+      errorPage(request, response, e.getMessage());
+    }
+  }
+
+  private void deleteProcess(HttpServletRequest request, HttpServletResponse response) {
+    try {
+      int id = parameterValueAsInt(request, 2);
+      employeeService.delete(id);
+      response.sendRedirect(request.getContextPath() + UrlConstants.EMPLOYEE_ROUTE);
+    }
+    catch (Exception e) {
+      LOGGER.log(Level.SEVERE, e.getMessage(), e);
+      errorPage(request, response, e.getMessage());
+    }
+  }
+
+  private void fetch(HttpServletRequest request, HttpServletResponse response) {
+    try {
+      Integer pageLimit = AppConstants.PAGE_LIMIT;
+      Integer pageNo = getCurrentPage(request);
+      Integer offset = (pageNo-1)*pageLimit;
+
+      List<Employee> employeeList = employeeService.fetch(pageLimit, offset);
+      Integer employeeCount = employeeService.count();
+      request.setAttribute(AttributeConstants.EMPLOYEE_LIST, employeeList);
+      request.setAttribute(AttributeConstants.EMPLOYEE_COUNT, employeeCount);
+      request.setAttribute(AttributeConstants.PAGE_NO, pageNo);
+      if ((employeeCount%pageLimit) == 0) {
```

Comment:
your code is not properly formatted. look at this statement there is no space after employeeCount and %